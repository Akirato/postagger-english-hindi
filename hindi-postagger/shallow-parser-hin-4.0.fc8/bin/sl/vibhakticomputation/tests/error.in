1	((	VGF	<fs af='Cipa,v,m,sg,any,,wA,wA' head="CipawA">
1.1	nahIM	NEG	<fs af='nahIM,adv,,,,,,' poslcat="NM">
1.2	CipawA	VM	<fs af='Cipa,v,m,sg,any,,wA,wA' name="CipawA">
1.3	hE	VAUX	<fs af='hE,v,any,sg,2,,hE,hE' name="hE">
	))
1	((	VGF	<fs af='Cipa,v,m,sg,any,,wA,wA' head="CipawA">
1.1	CipawA	VM	<fs af='Cipa,v,m,sg,any,,wA,wA' name="CipawA">
1.2	nahIM	NEG	<fs af='nahIM,adv,,,,,,' poslcat="NM">
1.3	hE	VAUX	<fs af='hE,v,any,sg,2,,hE,hE' name="hE">
	))
1	((	NP	<fs af='rAma,n,m,sg,,,,' head="rAma">
1.1	rAma	NN	<fs af='rAma,n,m,sg,,,,' name="rAma">
1.2	jI	RP	<fs af='jI,adv,,,,,,'>
1.3	ke	PSP	<fs af='ke,psp,,,,,,'>
1.4	sAWa	NST	<fs af='sAWa,adv,,,,,,'>
1.5	BI	RP	<fs af='BI,adv,,,,,,'>
	))
