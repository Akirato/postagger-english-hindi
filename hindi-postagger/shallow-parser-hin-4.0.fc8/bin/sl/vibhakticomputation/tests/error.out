1	((	VGF	<fs af='Cipa,v,m,sg,2,,wA_hE,wA' head="CipawA" vpos="tam2_3">
1.1	nahIM	NEG	<fs af='nahIM,adv,,,,,,' poslcat="NM">
1.2	CipawA	VM	<fs af='Cipa,v,m,sg,any,,wA,wA' name="CipawA">
	))		
2	((	VGF	<fs af='Cipa,v,m,sg,2,,wA_hE,wA' head="CipawA" vpos="tam1_3">
2.1	CipawA	VM	<fs af='Cipa,v,m,sg,any,,wA,wA' name="CipawA">
2.2	nahIM	NEG	<fs af='nahIM,adv,,,,,,' poslcat="NM">
	))		
3	((	NP	<fs af='rAma,n,m,sg,,,0_ke_sAWa,' head="rAma" vpos="vib1_RP_3_4_RP">
3.1	rAma	NN	<fs af='rAma,n,m,sg,,,,' name="rAma">
3.2	jI	RP	<fs af='jI,adv,,,,,,'>
3.3	BI	RP	<fs af='BI,adv,,,,,,'>
	))		
